app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,10 @@
     <string name="speech_button">Speech</string>
     <string name="listening_text_indicator">Listening!</string>
     <string name="not_listening_text_indicator">not listening...</string>
+    <string name="what">Confused?</string>
+    <string name="info_text">Want to hear what some random Redditor thinks about something- just ask Snoo about your topic! Ask Snoo Anything returns Reddit comments related to what you say.</string>
```

Comment:
grammar:

Want to hear what some random Redditor thinks about something? Just ask Snoo about your topic! *Ask Snoo Anything* returns Reddit comments related to what you say.